lib/src/runner/browser/content_shell.dart

```diff
@@ -59,9 +60,16 @@ class ContentShell extends Browser {
         // make sure it's not expired and that the remote debugging port worked.
         // Any errors from this will always come before the "Running without
         // renderer sandbox" message.
-        while (await stderr.moveNext() &&
+        while (await stderr
+                .moveNext()
+                .timeout(_errorTimeout)
+                .catchError((_) => false) &&
             !stderr.current.endsWith("Running without renderer sandbox") &&
-            !stderr.current.contains("Running without the SUID sandbox")) {
+            !stderr.current.contains("Running without the SUID sandbox") &&
+            // Error messages on Mac can get gobbled, we must assume that it
+            // it started up successfully.
```

Comment:
If it does report a "kqueue" message, can we just use that as a signal to stop listening rather than having a timeout?

Have you verified that it doesn't print anything if you're binding to a port that's definitely already allocated?